lib/promise.js

```diff
@@ -47,6 +50,13 @@ for (var method in Promise) {
 
 var bluebirdThen = Promise.prototype._then;
 Promise.prototype._then = function (didFulfill, didReject, didProgress, receiver, internalData) {
+  if (SequelizePromise.prototype.sequelize.options.namespace) {
```

Comment:
or `context`, although i suppose that's still as misleading if you don't know what CLS is.